cryptographic_estimators/PKEstimator/pk_algorithm.py

```diff
@@ -21,14 +21,13 @@
 
 class PKAlgorithm(BaseAlgorithm):
     def __init__(self, problem: PKProblem, **kwargs):
-        """
-        Base class for Permuted Kernel algorithms
-
-        INPUT:
+        """Base class for Permuted Kernel algorithms.
 
-        - ``problem`` -- LEProblem object including all necessary parameters
-        - ``cost_for_list_operation`` -- cost in Fq additions for one list operation in the SBC and KMP algorithms (default: n-m)
-        - ``memory_for_list_element`` -- memory in Fq elements for one list element in the SBC and KMP algorithms (default: n-m)
+        Args:
+            problem (PKProblem): LEProblem object including all necessary parameters
+            **kwargs: Additional keyword arguments
+                cost_for_list_operation (int): Cost in Fq additions for one list operation in the SBC and KMP algorithms (default: n-m)
+                memory_for_list_element (int): Memory in Fq elements for one list element in the SBC and KMP algorithms (default: n-m)
```

Comment:
wrong indentation

Reply:
To be fixed in #198

Reply:
Nothing to fix. Indentation is correctly parsed by our custom Sphinx extension (see https://github.com/Crypto-TII/CryptographicEstimators/pull/211).